src/main.rs

```diff
@@ -228,20 +230,11 @@ fn parse_failed<T>(key: &str, typename: &str) -> Option<T> {
 
 fn main() {
     // Load the user's config
-    let xdg = match xdg::BaseDirectories::new() {
-        Ok(xdg) => xdg,
-        Err(e) => {
-            println!("Could not find configuration file buzz.toml: {}", e);
-            return;
-        }
-    };
-    let config = match xdg.find_config_file("buzz.toml") {
-        Some(config) => config,
-        None => {
-            println!("Could not find configuration file buzz.toml");
-            return;
-        }
-    };
+    let config = ProjectDirs::from("jonhoo", "github", "buzz")
```

Comment:
We should make sure that, on Linux, this still ends up referring to the same path. Have you checked that?

The arguments here also don't seem quite right to me. I think the first two arguments should be the empty string, according to the docs over at https://docs.rs/directories-next/1.0.1/directories_next/struct.ProjectDirs.html#method.from.

Reply:
Ah, you're right. I wasn't aware that we could just pass the empty qualifier for the first two. I'll change that.
But yes, I've tested that this finds the same file on my Linux build.